src/plugins/discover/public/application/helpers/get_index_pattern_field_list.ts

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
It seems this file is leftover from some merge conflict or some previous trials? It is not used within this PR (and anyway a copy of another existing file).